### ISSUE TITLE
Infantry are bullet sponges

### ISSUE THREAD

**Issue by BlackAlpha** [acemod/ACE3#2624]
**ACE3 Version:**
3.3.2 Stable

**Mods:**
* `@CBA_A3`
* `@ace`

**Placed ACE3 Modules:**
None

**Description:**
Since version 3.3.0, all infantry seem to take less damage from bullets and so it takes a lot more bullets to take down infantry. This happens at all ranges but is the most noticeable at farther ranges.

It doesn't happen all the time because of the randomized damage, but it happens quite often. I've had cases where it took over 20 5.56mm (from a F2000) hits at 100 meters to take down an AAF rifleman.

Here's a video that clearly demonstrates the issue (targets are 300 meters away):
https://www.youtube.com/watch?v=vgp85YQStKM

**Where did the issue occur?**
Multiplayer on a dedicated server and in singleplayer.

**Steps to reproduce:**
Shoot at an infantry unit from at least 300 meters away. Try to hit the limbs. Notice how it takes a ridiculous amount of hits to make the AI stop shooting at you (ie. go unconscious permanently or die).

Here's a test mission in which you can reproduce the issue easily:
https://www.dropbox.com/s/gp98ix0ur1xmi07/T1_ShootingRange_ACE_v2.VR.rar?dl=0

Here's a mod that will remove sway and increase accuracy on the F2000 to make it easier to test and debug this issue:
https://www.dropbox.com/s/fo4exf00lrh2f0l/t1_perfectAim_vanilla.pbo?dl=0

**Comment by commy2** [comment]
Hello,
1. What Version of the game are you using?
2. Please upload the RPT.

**Comment by BlackAlpha** [comment]
Latest stable.

https://www.dropbox.com/s/qih5i2dqfz7rup7/arma3_2015-09-27_07-25-32.rar?dl=0

**Comment by 50KAL0V** [comment]
Have also experienced this with A3, CBA + ACE 3.3.2 on FIA units firing Stanag 556 at them (BIS mags not ACE). Also tried 762 and had similar results. Acting like they have too much hitpoints or the bullets arn't doing enough damage. 

For me it only seems to happen when I put a mission on dedicated server???

**Comment by alganthe** [comment]
CBA v2.0 and ACE3.3.2 no modules placed, FIA units die in one shot from a TRG 5.56 when hit in the torso.

**Comment by bux578** [comment]
@alganthe Did you test it on a dedicated server?

**Comment by alganthe** [comment]
nope @bux, I'll run one later today, i'll report back.

**Comment by BlackAlpha** [comment]
The issue is not with the torso but the arms and legs.

You need to shoot at a lot of units because the randomized damage will sometimes kill units with just one hit anywhere you hit. So, to get a baseline, you need to kill many AI.

**Comment by alganthe** [comment]
CBA_A3 v2.0 and ACE3.3.2 used.
no modules placed.

tests are done shooting the right leg at point blank (~20m) with the TRG using vanilla 5.56
 in editor, will try on dedi later

| test number | bullets fired |  unconscious after |
---------------- | --- |   ------------------------------------ |
1 | 2 | 1 |
2 | 5 | didn't fell unconscious |
3 | 5 | 3 |
4 | 3 | didn't fell unconscious |
5 | 3 | 1 |
6 | 1 | didn't fell unconscious |
7 | 3 | 1 |
8 | 3 | didn't fell unconscious  |
9 | 1 | didn't fell unconscious  |
10 | 4 | 2 |

hardly bullet sponges, will try on dedi later

**Comment by BlackAlpha** [comment]
Keep in mind that during a real mission you will be hitting the enemy all over the place. So, I suggest you do what I did in the video. Use my test mission, use my test mod and you will have an accurate test environment that is similar to a real mission in which you engage targets from further away and hit them on different body parts.

So, shoot one bullet at the leg, shoot another one at the arm, shoot another one at the torso, shoot another one at the first leg, etc. Then you'll see that all the bullets add up and you end up with bullet sponges.

**Comment by alganthe** [comment]
the TRG has a VERY short barrel and you're shooting over it's effective range, no surprises that the velocity of the bullet would be so low that it doesn't do that much damage.

**Comment by BlackAlpha** [comment]
So, if in real life you get hit by a 5.56 from 300 meters, and you get hit in your arm or leg, you will take negligible damage?

**Comment by alganthe** [comment]
correction, looked it up, the effective range is 550m (non certified), remember this is a game and everything might not be modeled 1:1, i'll check if the barrel has the correct length

**Comment by BlackAlpha** [comment]
Remember that the randomized damage will sometimes even kill with 1 hit to the leg or arms. So, I think ACE is trying to make things similar to real life where even a single bullet is sometimes enough to make someone combat ineffective. Maybe the issue is that the randomized damage sometimes does too little damage maybe?

**Comment by alganthe** [comment]
We need @glowbal and @ulteq inputs on this one.

**Comment by BlackAlpha** [comment]
I updated the links in the first post. The shooting range now has FIA soldiers. The accuracy mod now supports the Tavor (the one the FIA rifleman uses).

I get the same issue. Torso damage seems fine. Arms and legs I sometimes hit with 4 or more bullets in total and sometimes it goes much higher.

**Comment by BlackAlpha** [comment]
So, even if you consider it to be normal to have to hit someone 6-8 times on average to make them combat ineffective (which is about 2-3 more than in vanilla), there's still the issue that sometimes it takes a huge amount of hits to take someone out. I hope that at least that would be acknowledged. I wish to remind you that in the ending of the video I posted you can clearly see that it took 14 hits to take down one soldier, which clearly shows that something isn't working correctly:
https://youtu.be/vgp85YQStKM?t=2m34s

**Comment by looterz** [comment]
Been experiencing the same issue with ACE and RHS units & weapons, haven't tested with vanilla content yet but I'd imagine i would get similar results.

**Comment by vlad8011** [comment]
Cannot reproduce..... wirh RHS, CBA, ACE, Bcombat enemy infantry falls after max 5 hits

**Comment by looterz** [comment]
Did some more testing over the night with someone else and it seems to only occur on dedicated servers when shooting AI in the limbs or head and seems to depend on the weapon used and the range, we had ACE loading last on the client and server. 

The weapon I noticed it with most during testing was the Lee Enfield from the weapon community upgrade project, 1-2 shot kill from any range under 600m in SP, sometimes 10 or more in MP.

**Comment by bux578** [comment]
Please focus testing on vanilla units/weapons.

**Comment by looterz** [comment]
It looks like it would be very helpful for anyone testing to enable ACE_DEBUG and post logs from the test session as well.

**Comment by BlackAlpha** [comment]
OK. I got definite proof now! I knew I wasn't crazy... lol

I uploaded a new shooting range mission (see first post). It now shows you how many bullets it took to take down 1 soldier. And additionally, in your action menu you now have an action to show you this number for all downed soldiers and it will show you an overall average of rounds it took you in total. So, you can get concrete data that doesn't lie!

The mission can be run in singleplayer and on a dedicated server. I did a few runs on the dedicated server and I didn't see a difference between singleplayer and multiplayer; it takes the same amount of bullets on each.


Test instructions:
- Go to the first post and download version 2 of the shooting range. Load it up in the editor.
- Shoot each soldier on the left arm, then the torso, then the right arm, then the torso, then the left arm, repeat.
- Stop shooting the target when he goes down and then move on to the next target.
- You must down ALL 80 targets.
- When done, take a look at who stood back up and take them out again.
- When all targets are down, use the scroll wheel and select the action called "print bulletsarray" to see the average of the amount of bullets it took you to down a soldier.
- Repeat the test at least one more time, then take the average of your test results as the final result.


Results:

Arma3 v1.50
ACE3 v3.3.2
4.9 (singleplayer)
4.7 (singleplayer)
4.9 (Dedicated)
Average: 4.83 bullets on average to down a single soldier

Arma3 v1.50
ACE v3.2.1
2 (Dedicated)
2 (Dedicated)
2 (Dedicated)
Average: 2 bullets on average to down a single soldier

I did some additional testing and with ACE v3.2.1 it didn't matter where you hit them, they would almost always go down with 2 hits whether it was the arms or legs, as should happen in real life.


Conclusion:
In ACE v3.3.2 it takes about 60% more bullets to take down a regular soldier. And this is under ideal conditions when you are hitting the upper torso area and the arms. As I tried to explain before, if your bullets land at the legs as well, than it can take in v3.3.2 hell of a lot of bullets in total to take down a single soldier. So, if we compare v3.2.1 and v3.3.2, in v3.3.2 the enemies have been turned into bullet sponges, which is highly unrealistic and not fun at all.

**Comment by Glowbal** [comment]
Hello,

I've yet to try out your repro mission but I will get around to it eventually. Meanwhile, could you please provide me with some information on the medical settings you use?

Thanks in advanced.

**Comment by alganthe** [comment]
Note: no need to go and check your modules one by one, go into ace settings and press the debug to button, then, paste the result into a pastebin and give us the link :D

**Comment by BlackAlpha** [comment]
I didn't put any modules in the mission file. So, it's the default medical settings, whatever they are (I'm not sure what the default settings are).

**Comment by BlackAlpha** [comment]
I did another test. Same test conditions as above except I shot the targets as follows: left arm, left leg, right leg, right arm, repeat. I only did the test once, though, so it's not super accurate but good enough to show the differences. It was done in singleplayer.

Arma3 v1.52
ACE3 v3.3.2
Average: 5.4
A large chunk of the soldiers (about 25%) took around 10 hits.

Arma3 v1.52
ACE3 v3.2.1
Average: 1.9
Highest was 3 hits and it happened only with a few soldiers.

**Comment by ManOfHonor** [comment]
I conducted the same test that BlackAlpha did earlier.

* 300m
* FN2000 5.56mm
* AFF soldiers as targets
* Version 2 of the shooting range provided above. 
* Default ACE3 medical settings
* Extended Armor Disabled


Loaded it up in the editor. Shot each soldier on the left arm, then the torso, then the right arm, then the torso, then the left arm, repeat. Only shot in the torso and arms. Stopped shooting the target when he went down and then moved on to the next target. Downed all 80 targets. Shot the ones who went unconscious and woke up with the same method.

ARMA3 v1.52
ACE3 v3.3.2
One test, eighty targets
[4,6,3,4,5,8,4,7,5,4,6,4,5,4,4,4,4,4,1,4,4,4,4,1,5,6,5,1,3,4,5,4,7,1,5,3,6,4,4,1,3,4,6,5,4,4,4,4,5,5,4,4,4,4,5,3,4,5,4,6,7,5,8,5,5,8,3,8,7,6,5,6,5,8,6,5,6]
Average: 4.6
Enemies which took seven or eight bullets were relatively common. Even an average of four bullets represents a significant amount of trauma which the enemies are apparently shrugging off. Getting shot three times is nothing to sneeze at.


ARMA3 v1.52
ACE3 v3.2.1
One test, eighty targets
BulletArray- [2,3,2,2,2,2,2,2,2,2,1,2,2,2,1,2,2,2,1,2,1,3,2,2,2,2,2,2,2,2,1,2,1,1,1,1,2,2,2,1,1,2,1,3,1,1,3,2,2,2,1,2,2,1,2,2,2,3,2,2,2,2,3,2,2,2,2,2,2,2,2,3,2,2,2,2,3,1,1,1]
Average: 1.9
As you can see the most shots any target withstood while still remaining combat effective was two. Three bullets had a zero percent survival rate.


Conclusion: Enemies in ACE3 3.3.2 are, on average, more than twice as durable than in ACE3 3.2.1

**Comment by Kilo1-1** [comment]
This is related to #2573 I am sure.  In the meantime, I would recommend you adjust your AI damage values to be .5 or lower depending on your liking to how many shots should down an AI.

**Comment by commy2** [comment]
ACE v3.2.1 is not build for Arma v1.52 or v1.50 for that matter. It is based on the hitpoints configuration of v1.48 and below which was fixed in ACE v3.2.1.
For this test to work you have to compare:
v1.48 and ACE 3.1.whatever the latest was
with
v1.52 and ACE 3.2.1

**Comment by BlackAlpha** [comment]
@Kilo1-1 I tried that but it doesn't solve the issue. I did a quick test with 45 targets, setting the AI damage value to 0.5, shooting only arms and torso and this is the number of hits I got:

Number of hits / number of soldiers
1 = 17
2 = 4
3 = 4
4 = 1
5 = 7
6 = 3
7 = 3
8 = 1
9 = 4

That's still a lot of soldiers that take too many hits. It seems that there isn't really a way to get lower hit numbers because ACE's randomization makes it go all over the place.


@commy2 I see. But there isn't a way to rollback to that Arma version, is there? So, there's no way to check?

Well, we could compare ACE to vanilla but vanilla has always been super bullet spongy and BIS isn't that concerned with realism. Regardless, I did a full test shooting arms and torso and got an average of 5.58, with 6 being the highest. So, it's still quite different from ACE's enemies that can soak up around 10 hits or more.

I think that if you want to use vanilla as a baseline and you want to randomize it more, then it should not go higher than 7 (6 is already quite high!). I suspect that if you make ACE's current system less random than it is now, then people could more easily tweak it themselves (using the medical settings or tweaking the damage on the weapons, etc) because they will get more consistent results, and so they will be able to avoid the 6+ hits that are very common at the moment.

**Comment by bux578** [comment]
@BlackAlpha 
>But there isn't a way to rollback to that Arma version, is there? So, there's no way to check?

https://community.bistudio.com/wiki/Arma_3_Steam_Branches

**Comment by BlackAlpha** [comment]
@bux578 Only 1.50 is available to me.

**Comment by vlad8011** [comment]
You can go to any version by enering code "Arma3LegacyXXX" in the place where you are activating release candidate, and changing to dev version. In XXX place put number of version without dot.

**Comment by BlackAlpha** [comment]
I thought that only works to go back one version.

When I enter Arma3Legacy148 I get "beta access code invalid".

**Comment by jonpas** [comment]
1.48 Legacy Branch is now on 1.50 probably (due to stable being 1.52). `Arma3Legacy150` is the new one.

**Comment by BlackAlpha** [comment]
So, should I test it on legacy 1.50? I was under the impression that version is not the same as the old 1.48 stable.

**Comment by jonpas** [comment]
Well I don't know, I just know that `Arma3Legacy148` doesn't exist anymore.

**Comment by headedhorseman101** [comment]
With out being rude, the onus is on the ACE team. If damage values were updated, it is the ACE team's job to fix it because your mod modifies the damage system. I don't see the logic in saying that his test is invalid because it was not in the right version. You should be aiming for the most recent Arma 3 version, not a backlogged one.

**Comment by headedhorseman101** [comment]
Also, suggest you remove the can't reproduce tag as people have been able to reproduce it on multiple occasions

**Comment by headedhorseman101** [comment]
It still needs to be fixed, though. The random damage values is extremely erratic if it takes between 1-8 bullets to down a vanilla unit regardless of armor.

**Comment by headedhorseman101** [comment]
If you pass it off as Bis' fault, it will never get fixed.

**Comment by commy2** [comment]
Thats an assumption and is wrong. We are waiting until BI finishes their changes to soldier protection:
https://forums.bistudio.com/topic/160329-soldier-protection-dev-branch/?p=2916523

**Comment by headedhorseman101** [comment]
My apologies. Was not aware of that. Thank you for telling me.

**Comment by Maffaxxx** [comment]
@commy2 said: ACE v3.2.1 is not build for Arma v1.52 or v1.50 for that matter. It is based on the hitpoints configuration of v1.48 and below which was fixed in ACE v3.2.1.

But the only version we can roll back to is 1.50, which as per @commy2 words is not what ACE 3.2.1 nor 3.3.2 made for and wont budge the problem by one inch. 

So bottomline is that as things are now ACE is useless until BIS decides to complete the personal armor simulation, which can be next week or next month (or whenever). 

Am i correct?

And @commy2 why backrolling to 3.2.1 wouldnt solve a thing since as per @blackalpha simulation the results vary dramatically? What are your prediction on how this wouldnt work?

Thank you for your response :)

**Comment by SilentSpike** [comment]
Just to clarify, ACE 3.2.1 was made for Arma 3 1.50

**Comment by xJenny99x** [comment]
https://www.youtube.com/watch?v=0Vk8z3TFPBI

Is this part of the same issue? or is this intended?

**Comment by looterz** [comment]
Any updates on this issue? Been waiting on it to get fixed before diving back into ArmA 3.

**Comment by Glowbal** [comment]
This should be fixed. Any units that still are taking massive amounts of damage likely have non standard hitpoint configurations.

If this issue occures again, please create a new ticket.
